pkg/controller/nodeipam/config/v1alpha1/register.go

```diff
@@ -0,0 +1,31 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
```

Comment:
Since we're out-of-tree, I'm a bit unclear on what the stability graduation criteria is going to be. But I think keeping this as v1alpha1 makes sense.

@cheftako @bowei feel free to weigh in if you have any thoughts about stability level for the out-of-tree IPAM controller.

Reply:
I think its probably appropriate for now. Lets get it moved. Then clean it up for our use (If possible), then move it to Beta.